### DISCUSSION TITLE
Problem to start the game and resolution

### DISCUSSION THREAD

**Discussion by Marcharrierr** [PlayCover/PlayCover#1110] [Q&A] [answered]
I downloaded a game called Yugioh duel links, which when initialized, opens with the screen cut off and cannot be seen completely. When entering the graphics settings, I select the 4:3 aspect ratio or change it to something else and the game does not run. 

![Captura de pantalla 2023-09-29 a la(s) 18 17 50](https://github.com/PlayCover/PlayCover/assets/49212480/9b25d335-9543-47c9-8e07-9586d4ee8ad7)

Any solution?

**Comment by daoba2733** [comment]
自定义分辨率

**Reply by Marcharrierr** [reply]
What happens is that if I change the resolution, the game, I don't know why, but it doesn't open. When I click on the game, the game does not start and I have to download it again so it can start

**Comment by zanderp25** [comment]
I am able to reproduce this issue, and changing the resolution works, but does not seem to help

<img width="666" alt="image" src="https://github.com/PlayCover/PlayCover/assets/13000207/4833a9af-3419-40d1-bbb9-c667d1ed777a">
<img width="1008" alt="image" src="https://github.com/PlayCover/PlayCover/assets/13000207/c2781651-a89b-4012-a7f7-1de14e737b5d">

**Comment by zanderp25** [comment]
I have found a solution: change the Resolution Scaler to 1.0

<img width="1008" alt="Screenshot 2023-09-30 at 22 42 18" src="https://github.com/PlayCover/PlayCover/assets/13000207/bc29478a-fc03-4b85-8d00-d1bfb54e7bfa">
<img width="754" alt="Screenshot 2023-09-30 at 22 42 10" src="https://github.com/PlayCover/PlayCover/assets/13000207/ff563577-80d1-465a-a4d8-49132cc4f53a">

**Comment by TheMoonThatRises** [comment, marked as answer]
Change the app resolution to `App Default` and enable `Fix window display issues`.
